apps/gimbtonomy/GimbtonomyConfig.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include "ActuatorController.h"
+#include "Encoder.h"
+
+namespace GimbtonomyConfig {
+
+static Encoder::t_encoderConfig panEncoderConfig{
+    .pin_PWM = ENC_PWM_GIMB, .degreesPerUnit = 360.0, .zeroOffset_Degrees = 0.0};
+
+static ActuatorController::t_actuatorConfig panServoActuatorConfig = {
+    .defaultControlMode        = ActuatorController::position,
+    .minVelocity_DegreesPerSec = -264.0,
```

Comment:
Why are these limits so high?

Reply:
The limit is 44 RPM (no load), which os 264 deg/s
(https://www.robotshop.com/en/hitec-hsr-1425cr-continuous-rotation-servo.html)

Reply:
I see, just make sure the limit is realistic for the mechanism

Reply:
I think its fine to just use the no load value of the servo

Reply:
checked with mech, it should be around 38 RPM, I'll change it to that